### ISSUE TITLE
any plan to integrate Pinecone as an alternative vector db? 

### ISSUE THREAD

**Issue by prodonly** [Shaunwei/RealChar#53]
It would be easier to scale if hosted vector db can be used. 

Happy to contribute.

**Comment by pycui** [comment]
Yes it's on our roadmap, but not working on it currently. Contribution is very much welcome! Thanks
